samples/attested_tls/server/enc/server.cpp

```diff
@@ -127,9 +127,16 @@ int handle_communication_until_done(
 {
     int ret = 0;
     int len = 0;
+# ifdef CLOUD_DEPLOYMENT
```

Comment:
ifdefs are not as good as runtime options.  Recommend changing to use a command line argument to the sample.

Reply:
I agree with Dave.
To go even further, can you change the sample so that it just works for both the cases.
Can we not handle `MBEDTLS_ERR_SSL_CONN_EOF`  always rather than only enabling it for the autoscaling case?

Reply:
@anakrish  I hit this error when the TLS server received AKS load balancer pings. Not sure if there are other cases where this error is reported.

Reply:
Maybe we can handle the error always, with comments explaining the reasons and assumptions. Would that be acceptable?

Reply:
Yes. Handling that error always would be better.

Reply:
EOF errors are now always handled.